daliuge-translator/dlg/dropmake/lg.graph.schema

```diff
@@ -109,22 +124,12 @@
                     "collapsed": {
                         "type": "boolean"
                     },
-                    "streaming": {
-                        "type": "boolean"
-                    },
                     "subject": {
                         "type": "null"
                     },
                     "expanded": {
                         "type": "boolean"
                     },
-                    "precious": {
```

Comment:
**issue**: I think we still need precious here to preserve backwards compatibility with old graphs. IIRC this schema gets used to validate incoming JSON so it may return an error if an old chart is using the `precious` flag rather than allowing it and interpreting it as `persist`. Maybe could expand it with a description that it's different from the `fields` and `applicationArgs` version of `precious`

Reply:
I think this PR should remove "precious" from all the graphs in this repository.

Precious will still exist on many other graphs that originate from outside the repository, but in those cases the most likely way they are used is via EAGLE. When EAGLE loads a graph, then sends the graph for translation, the version sent for translation is automatically updated to the latest format, so it should not contain the "precious" attributes.

That leaves one final case, where graphs originate from outside this repo, but are input to the translator without EAGLE. I think in this case there will be a mismatch between the JSON and the schema. But I believe it won't return an error, since the "precious" attribute is not a "required" attribute.

Does this sound reasonable?

Reply:
That sounds reasonable to me. Might get @awicenec to drop in and see what he thinks - he wanted me to be able to handle the backwards compatibility in LIU-308 and will have a better idea of the usage patterns we can expect.

Reply:
That sounds good to me. Just a small wrinkle: when an old graph comes in with precious set to True, in principle this will need to be translated to the persist. If that proves hard to do I would just ignore it, since precious had not been used a lot at all.

Reply:
OK, I'll handle that on the EAGLE-side too